CareerPilot_Client/src/components/Navbar.jsx

```
@@ -167,15 +174,17 @@ const Navbar = () => {
-    {children}
-    <span className="absolute -bottom-1 left-0 w-0 h-0.5 bg-blue-500 transition-all duration-300 group-hover:w-full"></span>
-  </a>
-);
+const LinkStyled = ({ to, children }) => {
+  return (
+    <Link
+      to={to}
+      className="text-lg text-gray-300 cursor-pointer hover:text-white transition-all duration-200 relative group"
+    >
+      {children}
+      <span className="absolute -bottom-1 left-0 w-0 h-0.5 bg-blue-500 transition-all duration-300 group-hover:w-full"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)